src/request.js

```diff
@@ -0,0 +1,28 @@
+const request = new XMLHttpRequest();
+const requestURL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/9GFpJH58W0GP2nThYOqo/scores';
+
+async function getScores() {
+  const promise = new Promise((res) => {
+    request.open('GET', requestURL);
+    request.onload = () => {
+      if (request.status === 200) {
+        res(request.response);
+      } else {
+        res('Error');
+      }
```

Comment:
- [ ] For this project, You should use `fetch API` instead of `XMLHttpRequest`. (both of `getScores` and `scoreAddition` functions) Fetch API is more secure than XMLHttpRequest and it should be your first option. 

Please check [this article](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/articles/how_to_consume_api.md) for more information